src/ptr.ml

```diff
@@ -1,11 +1,15 @@
+[@@@coverage off]
 type ptr = Ligo.nat
 (* BEGIN_OCAML *) [@@deriving show] (* END_OCAML *)
+[@@@coverage on]
```

Comment:
Well, all derivings are OCaml-only so I guess we don't need these `BEGIN_OCAML`/`END_OCAML` markers here. This looks like a good opportunity to get rid of them :wink: 
```suggestion
[@@@coverage off]
type ptr = Ligo.nat
[@@deriving show]
[@@@coverage on]
```